iogt/templates/core/tags/section_listing_homepage.html

```diff
@@ -1,19 +1,55 @@
-{% load wagtailcore_tags static i18n %}
+{% load wagtailcore_tags wagtailimages_tags comments static core_tags i18n %}
 
 {% if sections %}
-<div id="section_menu" class="section-menu">
-  <div class="section-heading">
-    <a href="#"><img src="{% static 'img/iogt_logo_2.png' %}" class="logo" alt="The Internet of Good Things" width="40%" /></a>
-  </div>
-  <ul class="menu-list">
-    {% for section in sections %}
-    <li class="menu-list__item menu-list__item{{section.get_effective_extra_style_hints}}">
-      <a href="{% pageurl section %}">
-        {{section.title}}
-        <span class="menu-list__desc">{{section.description}}</span>
-      </a>
-    </li>
-    {% endfor %}
-  </ul>
-</div>
+<h3 class="heading heading--homepage">{% trans "Recommended" %}</h3>
+  {% for section in sections %}
+    {% load_descendant_articles_for_section section featured_in_homepage=True as featured_articles %}
+    {% if featured_articles %}
+    <div class="section-list section-list{{section.get_effective_extra_style_hints}}">
+      <ul class="article-list">
+      {% for article in featured_articles %}
+      <li class="article-list__item">
+      {% if forloop.first %}
+        <div class="article-list__content article-list__content--hero">
+          <a href="{% pageurl article %}" class="article-list__image">
+              {% if article.image %}
+                {% image article.image width-290 %}
+              {% endif %}
+          </a>
+          <h3 class="article-list__category">{{section.title}}</h3>
+          <a href="{% pageurl article %}" class="article-list__description">
+            <h2 class="article-list__title article-list__title--hero">{{article.title}}</h2>
+          </a>
+          <h4 class="article-list__subtitle">{{article.subtitle}}</h4>
+        </div>
+      {% else %}
+        <div class="article-list__content">
```

Comment:
I'm not a huge fan of this -- but if you feel that it's required then can we rename it to `article-list__container`